trail/trail.go

```diff
@@ -56,7 +56,7 @@ import (
 // * attaches debug metadata to existing metadata if possible
 // * sends the header to GRPC
 func Send(ctx context.Context, err error) error {
-	meta, ok := metadata.FromContext(ctx)
+	meta, ok := metadata.FromIncomingContext(ctx)
```

Comment:
I think you may need to be careful, as this is a breaking change in the gRPC API, and may not match the version of gRPC in our other projects.

Reply:
yes, Kevin, thanks for pointing this out. We do tag the releases - so this will be less of a problem if we depend on specific version of the package.